lib/service_extensions.dart

```diff
@@ -11,12 +11,16 @@ class ServiceExtensionDescription {
     this.extension,
     this.description,
     this.icon,
+    this.enabledValue = true,
```

Comment:
nit: it might be nice to make this class ServiceExtensionDescription<T>
where T is the type of the enabled and disabled value.

Reply:
done.